src/NewTwitchApi/Webhooks/WebhooksSubscriptionApi.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NewTwitchApi\Webhooks;
+
+use GuzzleHttp\Client;
+use NewTwitchApi\HelixGuzzleClient;
+
+class WebhooksSubscriptionApi
+{
+    public const SUBSCRIBE = 'subscribe';
+
+    private $clientId;
+    private $secret;
+    private $guzzleClient;
+
+    public function __construct(string $clientId, string $secret, Client $guzzleClient = null)
+    {
+        $this->clientId = $clientId;
+        $this->secret = $secret;
+        $this->guzzleClient = $guzzleClient ?: new HelixGuzzleClient($clientId);
```

Comment:
I would use `??` here since the default is `null` and to keep things consistent with https://github.com/ChessCom/twitch-api-php/pull/1/files#diff-904fe0dbc7f4ab3f08dfd68cbb1e0c0fR23